src/main/java/wolox/training/exceptions/ExceptionsResponse.java

```diff
@@ -0,0 +1,43 @@
+package wolox.training.exceptions;
+
+import java.sql.Timestamp;
+
+public class ExceptionsResponse {
+
+	public ExceptionsResponse(String error) {
+		this.error = error;
+	}
+
+	public ExceptionsResponse(String origin, String error) {
+		this.origin = origin;
+		this.error = error;
+	}
+
+	private String origin;
+	private Timestamp timestamp = new Timestamp(System.currentTimeMillis());
+	private String error;
```

Comment:
As a convention, we always declare all attributes at the top of the class, even before constructors 😋